src/DrawManager.sol

```diff
@@ -222,13 +220,15 @@ contract DrawManager {
    */
   function startDraw(address _rewardRecipient, uint32 _rngRequestId) external {
     if (_rewardRecipient == address(0)) revert RewardRecipientIsZero();
-    if (!_isNewDrawToAward()) revert CannotStartRngRequest();
+    uint24 drawId = prizePool.getDrawIdToAward(); 
+    if (prizePool.drawClosesAt(drawId) > block.timestamp) revert DrawHasNotClosed();
+    StartRngRequestAuction memory lastRequest = _lastStartRngRequestAuction;
+    if (lastRequest.drawId == drawId) revert AlreadyStartedDraw();
     if (rng.requestedAtBlock(_rngRequestId) != block.number) revert RngRequestNotInSameBlock();
 
     uint64 _auctionElapsedTimeSeconds = elapsedTimeSinceDrawClosed();
     if (_auctionElapsedTimeSeconds > auctionDuration) revert AuctionExpired();
```

Comment:
Should keep the `prizePool.drawClosesAt(drawId)` value to memory and use the internal `_elapsedTimeSinceDrawClosed(...)` call instead of the public one since the public one fetches the same data again.